CMakePresets.json

```diff
@@ -34,7 +34,54 @@
             "name": "ninja-vcpkg",
             "configurePreset": "ninja-multi-vcpkg",
             "displayName": "Build",
-            "description": "Build with Ninja/vcpkg"
+            "description": "Build with Ninja/vcpkg",
+            "hidden": true
+        },
+        {
+            "name": "ninja-vcpkg-check-format",
+            "configurePreset": "ninja-multi-vcpkg",
+            "displayName": "Build",
+            "targets": [
+                "check-format"
+            ],
+            "description": "Build with Ninja/vcpkg check-format"
```

Comment:
as in the `displayName`, is "build" appropriate? Could it be "Checking the format ..." or anything that is more accurate?